db/migrate/20171003182446_create_line_items.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLineItems < ActiveRecord::Migration
+  def change
+    create_table :line_items do |t|
+      t.integer :product_id
```

Comment:
It'd be great to add a not null constraint to these foreign keys.

Reply:
fixed